LoopKit/TemporaryScheduleOverrideSettings.swift

```diff
@@ -41,12 +45,19 @@ extension TemporaryScheduleOverrideSettings: RawRepresentable {
     private enum Key {
         static let targetRange = "targetRange"
         static let insulinNeedsScaleFactor = "insulinNeedsScaleFactor"
+        static let version = "version"
     }
 
     public init?(rawValue: RawValue) {
         if let targetRangeRawValue = rawValue[Key.targetRange] as? DoubleRange.RawValue,
             let targetRange = DoubleRange(rawValue: targetRangeRawValue) {
-            self.targetRange = targetRange
+            self.targetRangeInMgdl = targetRange
+        }
+        let version = rawValue[Key.version] as? Int ?? 0
+
+        // Do not allow target ranges from versions < 1, as there was no unit convention at that point.
+        if version < 1 && targetRange != nil {
```

Comment:
How prevalent is version (i.e. no version specified)? What is the side effect if the initializer returns `nil`?

Reply:
All override settings created prior to version 1 (jojo, mainly) will be wiped out, and the user will have to recreate them. This will have to be part of release notes for the next Loop release. This was also posted in Zulip for people switching to dev.

Reply:
:+1: